stable/neo4j/templates/readreplicas-deployment.yaml

```diff
@@ -24,6 +24,8 @@ spec:
         env:
           - name: NEO4J_dbms_mode
             value: READ_REPLICA
+          - name: NEO4J_ACCEPT_LICENSE_AGREEMENT
+            value: "yes"
```

Comment:
Same as my previous comment, make this a Chart variable that defaults to "no", so an end user has to explicitly accept the license.

Reply:
I would strongly recommend changing this to:
```yaml
          - name: NEO4J_ACCEPT_GPLv3_LICENSE_AGREEMENT
            value: "no"
```
Some enterprises refuse to use GPLv3. For example, Google treats GPLv3 in their `restricted` class: https://opensource.google.com/docs/thirdparty/licenses/

Helm Charts do not link with other software (like Helm), but it's still advisable to be as transparent as possible as to what the developer is agreeing to.